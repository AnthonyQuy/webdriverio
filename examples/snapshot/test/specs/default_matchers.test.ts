import { expect } from '@wdio/globals'

describe('Test default jest matchers', () => {
    it('should able to use default jest matchers', async () => {
        async function asyncFunction() {
            await new Promise(resolve => setTimeout(resolve, 1000))
            return { awaited: 'value' }
        }
        await expect(asyncFunction()).toMatchSnapshot()
        await expect({ a: 'aa' }).toMatchSnapshot()
        await expect({ a2: 'a' }).toMatchSnapshot('snapshot a2')
        await expect({ a2: 'a' }).toMatchInlineSnapshot()
    })
})
