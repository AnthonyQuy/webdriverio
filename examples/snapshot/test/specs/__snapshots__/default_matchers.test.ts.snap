// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test default jest matchers > should able to use default jest matchers 1`] = `
Object {
  "awaited": "value",
}
`;

exports[`Test default jest matchers > should able to use default jest matchers 2`] = `
Object {
  "a": "aa",
}
`;

exports[`Test default jest matchers > should able to use default jest matchers: snapshot a2 1`] = `
Object {
  "a2": "a",
}
`;
